cg/services/run_parameters_service/run_parameters_service.py

```diff
@@ -299,3 +311,13 @@ def get_read_1_cycles(self) -> int:
     def get_read_2_cycles(self) -> int:
         """Return the number of cycles in the second read."""
         return self._read_parser.get(RunParametersXMLNodes.READ_2_NOVASEQ_X)
+
+    def get_flow_cell_model(self) -> str:
+        """Return flow cell model."""
+        consumable_infos = self.tree.findall(".//ConsumableInfo")
+        for consumable_info in consumable_infos:
+            type_element = consumable_info.find("Type")
+            if type_element is not None and type_element.text == "FlowCell":
+                name_element = consumable_info.find("Name") or consumable_info.find("Mode")
```

Comment:
Is the "Mode" in this line necessary? Isn't the model of the X always under the field Name of the `FlowCell` node?

Reply:
No it can be "Name" or "Mode", beats me why they have two different types... but this is what I found accross the files. Maybe there is even another one. I am unsure at this point...